### PR TITLE
RedisGraph command compatibility

### PR DESCRIPTION
**Pull request by rrelledge**
[Staged preview](https://docs.redis.com/staging/jira-doc-1713/modules/redisgraph/commands/)

### CODE REVIEW

File: content/modules/redisgraph/commands.md [hunk 1]
```diff
@@ -10,14 +10,15 @@ categories: ["Modules"]
...
+| [GRAPH.CONFIG GET](https://redis.io/commands/graph.config-get/) | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Not supported"><nobr>&#x274c; Flexible & Annual</span><br /><span title="Not supported"><nobr>&#x274c; Free & Fixed</nobr></span> | Returns the current value of a [RedisGraph configuration parameter](https://redis.io/docs/stack/graph/configuration/#redisgraph-configuration-parameters). |
+| [GRAPH.CONFIG SET](https://redis.io/commands/graph.config-set/) | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Not supported"><nobr>&#x274c; Flexible & Annual</span><br /><span title="Not supported"><nobr>&#x274c; Free & Fixed</nobr></span> | Changes the value of a [RedisGraph configuration parameter](https://redis.io/docs/stack/graph/configuration/#redisgraph-configuration-parameters). |
+| GRAPH.DEBUG | <span title="Supported">&#x2705; Supported</span><br /><span><br /></span> | <span title="Not supported"><nobr>&#x274c; Flexible & Annual</span><br /><span title="Not supported"><nobr>&#x274c; Free & Fixed</nobr></span> | Debugging container command. |
```
Comment (LiorKogan) [inline content/modules/redisgraph/commands.md:17]: Please remove GRAPH.DEBUG it is an undocumented command

**Review by LiorKogan — COMMENTED** [review]
Please remove GRAPH.DEBUG

**Review by cmilesb — APPROVED** [review]
Looks good, with Lior's comment.

File: content/modules/redisgraph/commands.md (hunk 1 above)
Comment (rrelledge) [reply content/modules/redisgraph/commands.md]: Fixed.

**Pull request merged**